prisma/seeds/prices.ts

```diff
@@ -14,6 +14,45 @@ interface PriceFileData extends KeyValueT<string> {
   priceInUSD: string
 }
 
+export const mockPrices: Price[] = [
```

Comment:
This looks like the objects in `tests/mockedObjects.ts`, so I think it would better go there and then you can import it here

Reply:
done